tests/installation/agama_reboot.pm

```diff
@@ -38,9 +42,63 @@ sub get_agama_install_console_tty {
     return 7;
 }
 
+sub verify_agama_auto_install_done_cmdline {
+    # for some remote workers, there is no vnc access to the install console,
+    # so we need to make sure the installation has completed from command line.
+
+    # live root can support ssh access
+    my $timeout = 200;
+    while ($timeout > 0) {
+        if (script_run("journalctl -u agama | grep 'Install phase done'") == 0) {
+            record_info("agama install phase done");
+            return;
+        }
+        sleep 20;
+        $timeout = $timeout - 20;
```

Comment:
I would use sleep1 second and timeout timeout decrease by 1.

Reply:
I have seen that pattern before in our source code, basically it is for reducing resource usage, consumes less processing power and disk I/O compared to checking every second (in case of system has limited resources or if the log files are very large), also less noise when having some logging for each wait step in the loop. IMO should be fine 20.